xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -65,9 +75,19 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the collection of cipher suites passed through <see cref="M:System.Net.Security.CipherSuitesPolicy.#ctor(System.Collections.Generic.IEnumerable{System.Net.Security.TlsCipherSuite})" />, which are the ones allowed to negotiate with on a client or server <see cref="T:System.Net.Security.SslStream" /> authentication.</summary>
+        <value>The collection of cipher suites allowed in this policy for negotiation.</value>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[  
+> [!NOTE]
+> Defining a chipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to hold off the use of this property and rely on your constantly updated OS policy.
```

Comment:
Same comments as previous.

Reply:
```suggestion
> Defining a cipher suite policy on <xref:System.Net.Security.SslStream> authentication will prevent the OS of the ability to decide which are the best cipher suites to negotiate with and will need of your attention to manually check and update this code. It is strongly recommended to avoid the use of this property and rely on your constantly updated OS policy.
```